frontend/src/components/common/modal/Modal.tsx

```diff
@@ -38,6 +39,7 @@ const Modal = ({
   submitButtonVariant,
   cancelButtonVariant,
   onSubmit,
+  customBodyText,
```

Comment:
I'm making a PR for supporting toasts everywhere which has to add a bunch of props to this component... could we instead use the existing `body` attribute and allow rich components, so that the props don't grow out of hand here? See my next comment

Reply:
good idea